infrastructure/src/test/java/org/corfudb/infrastructure/management/FailureDetectorTest.java

```diff
@@ -46,7 +46,7 @@ public void testPollRound() {
         );
         Duration time = Duration.ofMillis(System.currentTimeMillis() - start);
         assertThat(time).isGreaterThan(Duration.ofMillis(450));
-        assertThat(time).isLessThan(Duration.ofSeconds(2));
+        assertThat(time).isLessThan(Duration.ofSeconds(20));
```

Comment:
Interesting, my patch is also failing on this test.

I feel like this number should be lower and perhaps fixed in a separate patch.

Reply:
Reverted back to 2. 2 is too short and it always fails for me locally.